ts/components/messages/paginated/MessagesInbox.tsx

```diff
@@ -62,31 +42,10 @@ const MessagesInbox = ({
     <View style={styles.listWrapper}>
       <View style={styles.listContainer}>
         <MessageList
-          onPressItem={(message: UIMessage) => {
-            if (selectedItems.isSome()) {
-              // Is the selection mode is active a simple "press" must act as
-              // a "longPress" (select the item).
-              toggleItem(message.id);
-            } else {
-              navigateToMessageDetail(message);
-            }
-          }}
-          onLongPressItem={toggleItem}
-          selectedMessageIds={selectedItems.toUndefined()}
+          onPressItem={(message: UIMessage) => navigateToMessageDetail(message)}
```

Comment:
```suggestion
          onPressItem={navigateToMessageDetail}
```
it's the same but with the suggestion we avoid the creation of a new function on each render

Reply:
Makes sense: https://github.com/pagopa/io-app/pull/3526/files#diff-83269738c78d711829f37c8f091c98a9eee02199bdfd8907c4cdfbcf581b3e2fR45